plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterPill.qml

```diff
@@ -12,7 +12,19 @@ import UM 1.2 as UM
 Item
 {
     // The printer name
-    property alias text: printerNameLabel.text;
+    property var text: ""
+    property var tagText: {
+        switch(text) {
+            case "Ultimaker 3":
+                return "UM 3"
+            case "Ultimaker 3 Extended":
+                return "UM 3 EXT"
+            case "Ultimaker S5":
+                return "UM S5"
+            default:
+                return ""
```

Comment:
Shouldn't it just return `text` if it is unknown?

Reply:
Mmmmm I guess it could. I didn't bother since we have only those 3 printers being showed in the UM3NetworkingPlugin but I guess might as well be ready for the next printer in the Ultimaker line up 😛

Reply:
Fixed